types/react-virtual-keyboard/index.d.ts

```diff
@@ -0,0 +1,31 @@
+// Type definitions for react-virtual-keyboard 1.0
+// Project: https://www.npmjs.com/package/react-virtual-keyboard
+// Definitions by: Bogdan Surai <https://github.com/bsurai>
```

Comment:
Please provide better documentation as without the npm package doesnt have source url and we wont be able to verify if you are writing correct definiton